src/hyperlight_guest_capi/README.md

```diff
@@ -1,4 +1,4 @@
-This is a c-api wrapper over the hyperlight-guest crate. The purpose of this crate is to allow the creation of guests in the c language. This crate generates a .lib/.a library file depending on the platform, as well necessary header files. 
+This is a c-api wrapper over the hyperlight-guest/hyperlight-guest-bin crate. The purpose of this crate is to allow the creation of guests in the c language. This crate generates a .lib/.a library file depending on the platform, as well necessary header files. 
```

Comment:
Maybe not something we want to do in this PR but should we create a C API over the new hyperlight-guest crate as well?